pom.xml

```diff
@@ -92,6 +92,10 @@
 
     <version.org.drools.guvnor-api>5.6.0.Final</version.org.drools.guvnor-api>
     <version.org.jboss.errai>4.0.1-SNAPSHOT</version.org.jboss.errai>
+
+    <!-- Required for kie-wb-common to build in IntelliJ Idea 2017 -->
+    <version.org.jboss.errai.wildfly>10.0.0.Final</version.org.jboss.errai.wildfly>
```

Comment:
This property is meant for [this](https://github.com/errai/errai/tree/master/errai-cdi/errai-wildfly10-dist) special version of wildfly-dist packaged in Errai. This wildfly-dist removes files in the normal wildfly-dist to workaround [this bug](https://youtrack.jetbrains.com/issue/IDEA-166417) in IntelliJ.